src/worker.js

```diff
@@ -92,6 +92,7 @@ const initServer = (app, config, callback) => {
   runAppLifecycle('initialize', config.plugins, config)
     .then(() => {
       server = app.listen(config.port, callback);
+      return null;
```

Comment:
You are right, returning something not consumed in the next `then` still makes more sense than simply returning null.

Fixed -> https://github.com/airbnb/hypernova/pull/41/commits/79c0dbb2727c1481d3190bc6bfebc4d4463096d0